metrics/api/src/main/java/io/helidon/metrics/api/MetricStore.java

```diff
@@ -67,7 +67,7 @@
     private final Map<String, List<MetricID>> allMetricIDsByName = new ConcurrentHashMap<>();
     private final Map<String, Metadata> allMetadata = new ConcurrentHashMap<>(); // metric name -> metadata
 
-    private final RegistrySettings registrySettings;
+    private RegistrySettings registrySettings;
```

Comment:
(`volatile`?)

Reply:
Not sure it's needed given how and from where the `update` method is invoked, but it's best to prevent any possible trouble. Good catch. Fixed.